FlagsmithClient/src/test/java/com/flagsmith/TraitsTests.kt

```diff
@@ -157,4 +183,20 @@ class TraitsTests {
             )
         }
     }
-}
\ No newline at end of file
+
+    @Test
+    fun testGetTransientIdentity() {
+        mockServer.mockResponseFor(MockEndpoint.GET_TRANSIENT_IDENTITIES)
+        runBlocking {
+            val result = flagsmith.getIdentitySync("transient-identity")
```

Comment:
This isn't testing transiency at all! It's not setting the `transient` attribute?

Reply:
I see that it's relying on the mocked server response, but really this isn't good enough.